room/compiler/src/test/data/databasewriter/output/ComplexDatabase.java

```diff
@@ -174,6 +177,14 @@ public void clearAllTables() {
         }
     }
 
+    @Override
+    protected Map<String, List<Class<?>>> getRequiredTypeConverters() {
+        final HashMap<String, List<Class<?>>> _typeConvertersMap =
+                new HashMap<String, List<Class<?>>>();
+        _typeConvertersMap.put("foo.bar.ComplexDao", ComplexDao_Impl.getRequiredConverters());
```

Comment:
Instead of using DAO class name in Strings we should probably use the Class itself to respect obfuscation.

Reply:
Right! Changed :)